gunicorn_config.py

```diff
@@ -0,0 +1,5 @@
+
+from gds_metrics.gunicorn import child_exit  # noqa
+
+worker_class = "eventlet"
+worker_connections = 256
```

Comment:
How does this affect performance as the there are 6 workers per paas instance?

Reply:
So with the eventlet worker_class it will do non-blocking io. So when it is talking to the database it will still serve other requests. The worker_connections limits the number of connections that each worker can do in this manner.

I can remove if you think there is something that this would interfere with.

Reply:
It is used on the paas already for notify. 

Something that walks through it's performance implications https://medium.com/@genchilu/brief-introduction-about-the-types-of-worker-in-gunicorn-and-respective-suitable-scenario-67b0c0e7bd62

Reply:
I've removed the worker_class eventlet as it was causing problems with the oidc client.